infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java

```diff
@@ -246,6 +246,12 @@ public void exitOnNoValidatorKeys() {
         "No loaded validators when --exit-when-no-validator-keys-enabled option is true. Shutting down...");
   }
 
+  public void validatorSlashedAlert(final Set<String> slashedValidatorPublicKeys) {
+    log.fatal(
+        "Validator(s) with public key(s) {} got slashed. Shutting down validator client...",
```

Comment:
this implies its only on VC? so if we're running a full BN/VC we can't use this new logic?

Reply:
Updated it: Removed `validator client`
It depends if the VC is running separately (in that case only the VC is shut down) or if VC/BN are running in a single process (in that case the Teku node is shut down)

Reply:
saying shutting down is what we generally say, thats fine.